certify-service/Dockerfile

```diff
@@ -45,8 +45,14 @@ ENV is_glowroot_env=${is_glowroot}
 # environment variable to pass artifactory url, at docker runtime
 ENV artifactory_url_env=${artifactory_url}
 
-# environment variable for certify artifactory plugins to pass at docker run time
-ENV enable_certify_artifactory="true"
+# environment variable for sunbird certify artifactory plugins to pass at docker run time
+ENV enable_sunbird_certify_artifactory="true"
+
+# environment variable for mosip identity certify artifactory plugins to pass at docker run time
+ENV enable_mosipid_certify_artifactory="false"
+
+# environment variable for mock identity certify artifactory plugins to pass at docker run time
+ENV enable_mockid_certify_artifactory="false"
```

Comment:
what will happen tomorrow a new plugin is created by a external team ? 
We cannot have any reference to mosipid/sunbird/mock in the certify code.

Reply:
this decision should be based on the config "mosip.certify.integration.vci-plugin"

Reply:
extracted all the plugins with single variable, so the implementation will be considered based on config properties